dist/templates/ovnkube-node.yaml.j2

```diff
@@ -39,7 +39,18 @@ spec:
       - name: ovnkube-node
         image: "{{ ovn_image | default('docker.io/ovnkube/ovn-daemonset:latest') }}"
         imagePullPolicy: "{{ ovn_image_pull_policy | default('IfNotPresent') }}"
-
+        {% if ovn_dummy_gateway_bridge=="true" %}
```

Comment:
ovnk is capable of creating the bridge. I don't think we need an extra dummy argument and bash code to do this. All we need to do is detect there is no gateway interface provided by config, and we detected no interface with a default gateway and then we create the bridge and assign some internal ip to it within ovnk.

Reply:
Currently, both LGW and SGW share the same logic of creating the gateway bridge, if `--gateway-interface` is empty. OVN-K will create a bridge with the gateway interface of the host. If we change the logic for LGW to what you described. Although, OCP will not be affected. I am concerned that it makes trouble for upgrading existing LGW deployments.

Reply:
This is for mimicking what we want to have in microshift. Instead of relying on ovn-k to create the dummy interface, I want to create the br-ex first, pass `--gateway-interface=br-ex` to ovnkube.